lib/api/api.dart

```diff
@@ -457,28 +445,13 @@ class ApiSingleton {
             'Request: ${response.request.toString()} -> Response: ${response.body}');
         return null;
       } else {
-        Map<String, dynamic> jsonAnswer = json.decode(response.body);
-        List<Report> list = [];
-        page = jsonAnswer['next'];
-
-        if (allReports == null) {
-          allReports = [];
-          UserManager.profileUUIDs = jsonAnswer['user_uuids'];
-        }
-        for (var item in jsonAnswer['results']) {
-          if (UserManager.profileUUIDs.any((id) => id == item['user'])) {
-            list.add(Report.fromJson(item));
-          }
-        }
-        allReports.addAll(list);
-        //callback(list);
-        if (page == null) {
-          return allReports;
+        List<dynamic> jsonAnswer = json.decode(response.body);
+        var list = <Report>[];
```

Comment:
`var List<Report> list = [];` ?

Reply:
This would get this error
```
Variables can't be declared using both 'var' and a type name.
Try removing 'var.'dart(var_and_type)
```

This would work: `List<Report> list = [];`
but then we get: 
```
Omit type annotations for local variables.dart
[omit_local_variable_types](https://dart-lang.github.io/linter/lints/omit_local_variable_types.html)
```

Reply:
You're right, i wasn't aware of that!